arduino/WandCore/CAPClient.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+  Capacitive Touch Client
+
+  Manages a Touch Surface Input
+*/
+
+#include "CAPClient.h"
+
+// touch threshold
+const int touchThold = 50; //above means not touched
+
+// Touch surface timing variables
+int debounce = 20;                // ms debounce period to prevent flickering when pressing or releasing the button
+int DCgap = 250;                  // max ms between clicks for a double click event
+int holdTime = 1000;              // ms hold period: how long to wait for tap+hold event

```

Comment:
My gut feeling says this can be a little shorter, but experimentation can resolve that!

Reply:
noted, no changes now